common/src/main/java/com/wynntils/functions/WorldFunctions.java

```diff
@@ -57,4 +57,56 @@ public List<String> getAliases() {
             return List.of("world_uptime", "uptime");
         }
     }
+
+    public static class MobTotemCountFunction extends Function<Integer> {
```

Comment:
I don't really think this fits in this file, but I have no better suggestion. We can let them live here for the time being, but I think long-term they need a better home.

Reply:
Yeah, I too did not know what do with this.